collector/generatorreceiver/internal/topology/file.go

```diff
@@ -1,10 +1,10 @@
 package topology
 
 type File struct {
-	Topology *Topology `json:"topology"`
+	Topology *Topology `json:"topology" yaml:"topology"`
```

Comment:
Why are we adding tags when this is the default expected key for yaml.Unmarshal?

Reply:
Valid question.  I did it just for "completeness" and to have parity with the `json:` annotations.